invenio_pages/alembic/98e0a418340d_create_invenio_pages_branch.py

```diff
@@ -0,0 +1,28 @@
+#
+# This file is part of Invenio.
+# Copyright (C) 2022 CERN.
+#
+# Invenio is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Create invenio_pages branch"""
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '98e0a418340d'
+down_revision = None
+branch_labels = ('invenio_pages',)
+depends_on = None
```

Comment:
question: other modules have `depends_on` a common revision [`dbdbc1b19cf2`](https://github.com/inveniosoftware/invenio-db/blob/9ea8eb3a6f05925d26dacf76645610bccef4f278/invenio_db/alembic/dbdbc1b19cf2_create_transaction_table.py#L16) from `invenio-db`. 

e.g.:
- [`invenio-rdm-records`](https://github.com/inveniosoftware/invenio-rdm-records/blob/8f2f01b8264d28e0bdfca85e592ec4ea05952c7c/invenio_rdm_records/alembic/b822ba22c688_create_rdm_records_branch.py#L18) 
- [`invenio-vocabularies`](https://github.com/inveniosoftware/invenio-vocabularies/blob/master/invenio_vocabularies/alembic/8ff82dfb0be8_create_vocabularies_branch.py#L18)

Is it relevant for this case or future migrations of `invenio-page`?

Reply:
Right, thanks!